docs/topics/git.md

```
@@ -0,0 +1,149 @@
+- git init
+- git add
+- git status
+- git log
+- git commit
+- git clone
+- git pull
+- git push
+
+### 😀 Git 버전 확인
+- Window는 git-bash에서 리눅스, Mac은 터미널에서
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)